exporter/logzioexporter/README.md

```diff
@@ -83,6 +165,3 @@ service:
     logs:
       level: debug #activate debug mode
 ```
-
-[stable]:https://github.com/open-telemetry/opentelemetry-collector#stable
```

Comment:
removing this breaks the link above (I think).  Can you add it back, and also add a link to beta (so the logs link works correctly).